scripts/build/create-scratch-org.ps1

```diff
@@ -29,22 +29,22 @@ Write-Output "Scratch org duration in days: $durationdays"
 
 foreach($devHub in $devHubs) {
     Write-Output "Trying dev hub: $devHub"
-    $scratchOrgAllotment = ((npx sfdx force:limits:api:display --json --targetusername $devHub | ConvertFrom-Json).result | Where-Object -Property name -eq "DailyScratchOrgs").remaining
+    $scratchOrgAllotment = ((npx sf limits api display --json --target-org $devHub | ConvertFrom-Json).result | Where-Object -Property name -eq "DailyScratchOrgs").remaining
     Write-Output "Total remaining scratch orgs for the day for $devHub : $scratchOrgAllotment"
 
     if($scratchOrgAllotment -gt 0) {
         Write-Output "Beginning scratch org creation"
         # Create Scratch Org
         try {
-            $scratchOrgCreateMessage = npx sfdx force:org:create --setdefaultusername --targetdevhubusername $devHub --durationdays $durationdays --definitionfile $definitionFile --wait 10
+            $scratchOrgCreateMessage = npx sf org create scratch --set-default --target-dev-hub $devHub --duration-days $durationdays --definition-file $definitionFile --wait 10
             # Sometimes SFDX lies (UTC date problem?) about the number of scratch orgs remaining in a given day
             # The other issue is that this doesn't throw, so we have to test the response message ourselves
             Write-Output "Scratch org creation mesage: $scratchOrgCreateMessage"
             if($scratchOrgCreateMessage -eq 'The signup request failed because this organization has reached its active scratch org limit') {
                 throw $1
             }
 
-            $defaultUsername = (npx sfdx config:get defaultusername  --json | ConvertFrom-Json).result.value
+            $defaultUsername = (npx sf config get target-dev-hub  --json | ConvertFrom-Json).result.value
```

Comment:
This should use `target-org` instead of `target-dev-hub`

```suggestion
            $defaultUsername = (npx sf config get target-org --json | ConvertFrom-Json).result.value
```